src/Computer.php

```diff
@@ -293,16 +294,43 @@ public function post_updateItem($history = 1)
     public function prepareInputForAdd($input)
     {
 
+        //without right, use can't set remote_addr
+        if (isset($input['remote_addr']) && !Session::haveRight(Conf::$rightname, Conf::IMPORTFROMFILE)) {
+            unset($input['remote_addr']);
+        }
+
```

Comment:
```suggestion
```

Done in `manageInput()` called at the end of the method, no need to unset it twice

Reply:
agree with this